htlcswitch/link.go

```diff
@@ -2192,6 +2193,31 @@ func (l *channelLink) MayAddOutgoingHtlc() error {
 	return l.channel.MayAddOutgoingHtlc()
 }
 
+type dustClosure func(chainfee.SatPerKWeight, bool, bool, btcutil.Amount) bool
```

Comment:
nit: add a comment explaining what the bools are in this function signature?

Reply:
Yes it's vague